src/Model/QuestionManager.php

```diff
@@ -11,4 +11,20 @@ public function selectRandomQuestion(): array
         $query = ("SELECT * FROM " . static::TABLE . " ORDER BY rand() LIMIT 1");
         return $this->pdo->query($query)->fetch();
     }
+
+    public function addQuestion(string $question, int $timeLimit): int
+    {
+        $query = "INSERT INTO " . static::TABLE . " (title, timelimit) VALUES (:question, :timelimit)";
+        $statement = $this->pdo->prepare($query);
+        $statement->bindValue(':question', $question, \PDO::PARAM_STR);
+        $statement->bindValue(':timelimit', $timeLimit, \PDO::PARAM_STR);
+        $statement->execute();
+        return $this->selectLastQuestion()["id"];
```

Comment:
It seems that you want to recreate this : https://www.php.net/manual/fr/pdo.lastinsertid.php
You probably should use it instead